2022-03-28 13:14:27,621 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-28 13:14:27,622 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-28 13:14:27,622 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-28 13:14:27,622 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-28 13:14:27,622 INFO    MainThread:12299 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_131427-3ragan0i/logs/debug.log
2022-03-28 13:14:27,622 INFO    MainThread:12299 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_131427-3ragan0i/logs/debug-internal.log
2022-03-28 13:14:27,622 INFO    MainThread:12299 [wandb_init.py:init():439] calling init triggers
2022-03-28 13:14:27,622 INFO    MainThread:12299 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-28 13:14:27,622 INFO    MainThread:12299 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 3qhszuzm
2022-03-28 13:14:27,624 INFO    MainThread:12299 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/3qhszuzm
2022-03-28 13:14:27,624 INFO    MainThread:12299 [jupyter.py:save_history():429] not saving jupyter history
2022-03-28 13:14:27,624 INFO    MainThread:12299 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-28 13:14:27,624 INFO    MainThread:12299 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-28 13:14:27,625 INFO    MainThread:12299 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-28 13:14:27,625 INFO    MainThread:12299 [wandb_run.py:_restore():1674] restore
2022-03-28 13:14:28,321 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 13:14:28,661 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 13:14:29,362 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 170338
}

2022-03-28 13:14:29,471 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 170338
}

2022-03-28 13:14:29,578 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 170338
}

2022-03-28 13:14:29,690 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 170338
}

2022-03-28 13:14:29,801 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170338
  total_bytes: 170338
}

2022-03-28 13:14:29,927 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170338
  total_bytes: 170338
}

2022-03-28 13:14:30,036 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170338
  total_bytes: 170338
}

2022-03-28 13:14:30,143 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170338
  total_bytes: 170338
}

2022-03-28 13:14:30,252 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170338
  total_bytes: 170338
}

2022-03-28 13:14:30,361 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170338
  total_bytes: 170338
}

2022-03-28 13:14:30,470 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170338
  total_bytes: 170338
}

2022-03-28 13:14:31,383 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170338
  total_bytes: 170338
}

2022-03-28 13:14:31,957 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170338
  total_bytes: 170338
}
local_info {
}

2022-03-28 13:14:33,192 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-28 13:14:33,194 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-28 13:14:33,197 INFO    MainThread:12299 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-28 13:14:33,199 INFO    MainThread:12299 [wandb_init.py:init():492] starting backend
2022-03-28 13:14:33,199 INFO    MainThread:12299 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-03-28 13:14:33,202 INFO    MainThread:12299 [backend.py:ensure_launched():219] starting backend process...
2022-03-28 13:14:33,231 INFO    MainThread:12299 [backend.py:ensure_launched():224] started backend process with pid: 16575
2022-03-28 13:14:33,232 INFO    MainThread:12299 [wandb_init.py:init():501] backend started and connected
2022-03-28 13:14:33,254 INFO    MainThread:12299 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-28 13:14:33,254 INFO    MainThread:12299 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-28 13:14:33,254 INFO    MainThread:12299 [wandb_init.py:init():565] updated telemetry
2022-03-28 13:14:33,338 INFO    MainThread:12299 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-28 13:14:34,548 INFO    MainThread:12299 [wandb_run.py:_on_init():1759] communicating current version
2022-03-28 13:14:34,591 INFO    MainThread:12299 [wandb_run.py:_on_init():1763] got version response 
2022-03-28 13:14:34,591 INFO    MainThread:12299 [wandb_init.py:init():625] starting run threads in backend
2022-03-28 13:14:36,838 INFO    MainThread:12299 [wandb_run.py:_console_start():1733] atexit reg
2022-03-28 13:14:36,838 INFO    MainThread:12299 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-28 13:14:36,839 INFO    MainThread:12299 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-28 13:14:36,839 INFO    MainThread:12299 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-28 13:14:36,839 INFO    MainThread:12299 [wandb_init.py:init():664] run started, returning control to user process
2022-03-28 13:14:36,839 INFO    MainThread:12299 [wandb_watch.py:watch():47] Watching
2022-03-28 13:21:54,217 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set batch_size = 50 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b3615b80>>
2022-03-28 13:21:54,217 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb batch_size 50 None
2022-03-28 13:21:54,217 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set epochs = 45 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b3615b80>>
2022-03-28 13:21:54,217 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb epochs 45 None
2022-03-28 13:21:54,217 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set lr = 4.674718736501256e-05 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b3615b80>>
2022-03-28 13:21:54,217 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb lr 4.674718736501256e-05 None
2022-03-28 13:21:54,218 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b3615b80>>
2022-03-28 13:21:54,218 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb no_cuda False None
2022-03-28 13:21:54,218 INFO    MainThread:12299 [wandb_config.py:__setitem__():153] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7f83b3615b80>>
2022-03-28 13:21:54,218 INFO    MainThread:12299 [wandb_run.py:_config_callback():992] config_cb seed 42 None
2022-03-28 13:22:19,493 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-28 13:22:19,494 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-28 13:22:19,494 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-28 13:22:19,494 INFO    MainThread:12299 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-28 13:22:19,494 INFO    MainThread:12299 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_132219-g5hqc59t/logs/debug.log
2022-03-28 13:22:19,494 INFO    MainThread:12299 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220328_132219-g5hqc59t/logs/debug-internal.log
2022-03-28 13:22:19,494 INFO    MainThread:12299 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f83b4112ee0>
2022-03-28 13:22:19,494 INFO    MainThread:12299 [wandb_init.py:init():439] calling init triggers
2022-03-28 13:22:19,494 INFO    MainThread:12299 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-28 13:22:19,494 INFO    MainThread:12299 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 3ragan0i
2022-03-28 13:22:19,499 INFO    MainThread:12299 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/3ragan0i
2022-03-28 13:22:19,499 INFO    MainThread:12299 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-28 13:22:19,503 INFO    MainThread:12299 [wandb_run.py:_restore():1674] restore
2022-03-28 13:22:20,035 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 13:22:20,280 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 824
}

2022-03-28 13:22:20,421 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 164605
}

2022-03-28 13:22:20,530 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 170501
}

2022-03-28 13:22:20,637 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 824
  total_bytes: 170501
}

2022-03-28 13:22:20,745 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170501
  total_bytes: 170501
}

2022-03-28 13:22:20,855 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170501
  total_bytes: 170501
}

2022-03-28 13:22:20,983 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170501
  total_bytes: 170501
}

2022-03-28 13:22:21,093 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170501
  total_bytes: 170501
}

2022-03-28 13:22:21,205 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170501
  total_bytes: 170501
}

2022-03-28 13:22:21,312 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170501
  total_bytes: 170501
}

2022-03-28 13:22:21,419 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170501
  total_bytes: 170501
}

2022-03-28 13:22:21,533 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170501
  total_bytes: 170501
}

2022-03-28 13:22:22,615 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170501
  total_bytes: 170501
}

2022-03-28 13:22:23,242 INFO    MainThread:12299 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 170501
  total_bytes: 170501
}
local_info {
}

2022-03-28 13:22:24,464 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-28 13:22:24,467 INFO    MainThread:12299 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-28 13:22:24,475 INFO    MainThread:12299 [wandb_run.py:_footer_sync_info():2822] logging synced files
